packages/transport-ble/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@aries-framework/transport-ble",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.1.0",
+  "files": [
+    "build"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "description": "Bluetooth Low Energy transport for Aries Framework JavaScript",
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript-ext/tree/main/packages/transport-ble",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript-ext",
+    "directory": "packages/transport-ble"
+  },
+  "scripts": {
+    "sample": "tsnd --respawn samples/sample.ts",
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf -rf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@animo-id/react-native-ble-didcomm": "^0.4.1",
+    "@aries-framework/core": "^0.3.2",
+    "react-native": "^0.70.6"
```

Comment:
I've moved them to `devDependencies`, so that the needed type definitions are there during development, but they function in production as `peerDependencies`